src/main/java/org/home/QueueApplication.java

```diff
@@ -0,0 +1,13 @@
+package org.home;
+
+import org.home.adapter.QueueAdapter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class QueueApplication {
```

Comment:
Let the application be starting on a `port` which is passed as commandLine argument to the application, not read from application.properties file. 
In this way, we will be able to test multiple instances of the process running on multiple port at the same time. 

This is feature ask